buildSrc/src/main/java/UpdateChangelogTask.kt

```diff
@@ -0,0 +1,42 @@
+import kotlinx.serialization.json.Json
+import org.gradle.api.DefaultTask
+import org.gradle.api.tasks.Input
+import org.gradle.api.tasks.TaskAction
+import org.intellij.markdown.flavours.commonmark.CommonMarkFlavourDescriptor
+import org.intellij.markdown.html.HtmlGenerator
+import org.intellij.markdown.parser.MarkdownParser
+import java.io.File
+
+/** Convert a markdown changelog to HTML. */
+open class UpdateChangelogTask : DefaultTask() {
+
+    @get:Input lateinit var sourceFile: String
+    @get:Input lateinit var targetFile: String
+
+    private val markdownFlavour = CommonMarkFlavourDescriptor()
```

Comment:
Can we use Github flavour? I post the current changelog also for each result, so it would be nice if I could use all the markdown that is supported by github

Reply:
Yes, it's easy to switch to `GFMFlavourDescriptor`, but it only changes one thing in the output: `www.AddisMap.com` (from [v33.2](https://github.com/streetcomplete/StreetComplete/releases/tag/v33.2)) is rendered as a link instead of plaintext, because GFM linkifies URLs, while CommonMark does not.

Reply:
That's fine, it's for the future, too

Reply:
Okay, changed to GitHub Flavored Markdown in [`f20d12b`](https://github.com/streetcomplete/StreetComplete/pull/5439/commits/f20d12b5ce78213e9ec9e21844f88aafd726bcf6).